mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule ExSamples.Mixfile do
     [
       app: :exsamples,
       version: "0.1.0",
-      elixir: "~> 1.5",
+      elixir: "~> 1.13",
```

Comment:
```suggestion
      elixir: "~> 1.13.2",
```

As that's actually the min version that [allows plugins to format `.ex` and `.exs` files](https://hexdocs.pm/elixir/1.13/changelog.html#v1-13-2-2022-01-13).